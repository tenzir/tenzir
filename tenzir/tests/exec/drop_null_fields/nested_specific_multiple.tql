// test: exec

// Test drop_null_fields with multiple specific fields containing nested nulls
from {
  a: {x: null, y: 1},
  b: {p: null, q: null},
  c: {m: 2, n: null}
}
drop_null_fields a, b