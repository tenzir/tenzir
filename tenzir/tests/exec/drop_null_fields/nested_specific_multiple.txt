{
  a: {
    y: 1,
  },
  b: {},
  c: {
    m: 2,
    n: null,
  },
}
