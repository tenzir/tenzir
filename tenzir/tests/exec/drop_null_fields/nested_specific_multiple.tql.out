{
  a: {
    y: 1,
  },
  c: {
    m: 2,
    n: null,
  },
}
